saleor/tax/migrations/0004_migrate_tax_classes.py

```diff
@@ -107,6 +111,35 @@ def migrate_product_tax_codes(apps, _schema_editor):
             Product.objects.filter(id__in=ids).update(tax_class=tax_class)
 
 
+def migrate_products_with_disabled_taxes(apps, _schema_editor):
+    Product = apps.get_model("product", "Product")
+    TaxClass = apps.get_model("tax", "TaxClass")
+    TaxClassCountryRate = apps.get_model("tax", "TaxClassCountryRate")
+
+    qs = Product.objects.filter(charge_taxes=False).order_by("pk")
+    zero_rate_tax_class, _ = TaxClass.objects.get_or_create(
```

Comment:
Is it expected that you will always create the zero rate tax class? Shouldn't we create it only when we have at least one product with charge_taxes==False?

Reply:
Fixed